website/docs/r/default_internet_gateway.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "aws"
+page_title: "AWS: aws_default_internet_gateway"
+sidebar_current: "docs-aws-resource-default-internet-gateway"
+description: |-
+  Provides a resource to manage a default VPC Internet Gateway.
+---
+
+# aws_default_internet_gateway
+
+Provides a resource to manage a [default VPC Internet Gateway](https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/default-vpc.html#default-vpc-components)
```

Comment:
Do you think we should provide a more strict warning, akin to https://www.terraform.io/docs/providers/aws/r/default_route_table.html, about the fact that Terraform cannot destroy this resource?

Reply:
@jen20 I think that's already taken care of in the _Removing aws_default_internet_gateway from your configuration_ section below.